draft-ietf-lake-edhoc.md

```diff
@@ -1796,11 +1801,16 @@ This sections compiles the CDDL definitions for ease of reference.
 ~~~~~~~~~~~ CDDL
 suites = [ 2* int ] / int
 
-ead = 1* (
+ead = (
   ead_label : int,
-  ead_value : bstr,
+  ? ead_value : bstr,
 )
 
+EAD_1 = 1* ead
```

Comment:
Why do you define EAD_x and do not use it here?

Reply:
Ambivalence. Happy to change to

> EAD_x = 1* ead

How would CDDL handle the part:

> for x = 1, 2, 3, 4 

?

Reply:
CDDL has generics for this, but in this case you just want to ignore the 1, 2, 3, 4?

```
EAD_<x> = (+ ead)
... EAD_<1> ...
```

Not sure why these gymnastics are needed.